readthedocs/doc_builder/environments.py

```diff
@@ -1039,9 +1049,11 @@ def create_container(self):
                 ),
                 name=self.container_id,
                 hostname=self.container_id,
+                volumes=self._get_binds(),
                 host_config=self.get_container_host_config(),
                 detach=True,
                 environment=self.environment,
+                user=settings.RTD_DOCKER_USER,
```

Comment:
I worry a bit about this in prod, any reason this is needed?

Reply:
Yes. This avoid permission issues. There more context in this PR https://github.com/readthedocs/readthedocs.org/pull/5556

This won't affect production since we are defining `DOCKER_USER = 'docs:docs'` on default settings (https://github.com/readthedocs/readthedocs.org/pull/6295/files#diff-5973fe59a4d52278d55f884d145bd5d7R357) --which is the user and group used by default if this attribute is not specified.